src/reanimated2/layoutReanimation/animationBuilder/Keyframe.ts

```diff
@@ -0,0 +1,199 @@
+import { Easing, EasingFn } from '../../Easing';
+import { withDelay, withSequence, withTiming } from '../../animations';
+import {
+  AnimationFunction,
+  EntryExitAnimationBuild,
+  IEntryExitAnimationBuilder,
+  KeyFrameProps,
+  StyleProps,
+} from './commonTypes';
+
+export interface KeyFramePoint {
```

Comment:
Naming is inconsistent: this is named `KeyFramePoint` and class is named `Keyframe`. I personally prefer `Keyframe` prefix.